client/src/js/analyses/components/Pathoscope/__tests__/Toolbar.test.js

```diff
@@ -1,3 +1,4 @@
+import { DropdownItem } from "../../../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['DropdownItem' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5050628)